src/pages/student_fees_list/FeesList.tsx

```diff
@@ -0,0 +1,277 @@
+  );
+  const [statusFilter, setStatusFilter] = useState("");
+  const [modalType, setModalType] = useState<"pay" | "details" | null>(null);
+  const [selectedFeeId, setSelectedFeeId] = useState<string | null>(null);
+  const [selectedPaymentId, setSelectedPaymentId] = useState<string | null>(
+    null
+  );
+  const [payMonthlyFee] = useUpdateMonthlyFeeMutation();
+
+  const normalizeDate = (date: Date | string | null) => {
+    if (!date) return undefined;
+
+    const d = new Date(date);
+
+    if (isNaN(d.getTime())) return undefined;
+    return d.toISOString().split("T")[0]; // "YYYY-MM-DD"
+  };
+  const formattedExpirationDate = normalizeDate(expirationDateFilter);
+  const {
+    data: fees,
+    isLoading,
+    isError,
+  } = useGetStudentMonthlyFeesQuery(
+    student?.id
+      ? {
+          studentId: student.id,
+          month: monthFilter || undefined,
+          expirationDate: formattedExpirationDate,
+          status: statusFilter || undefined,
+        }
+      : skipToken
+  );
+
+  const handleOpenPayModal = (feeId: string) => {
+    setSelectedFeeId(feeId);
+    setModalType("pay");
+  };
+  const handleOpenPaymentInfoModal = (paymentId?: string) => {
+    setSelectedPaymentId(paymentId!);
+    setModalType("details");
+  };
+  const handleConfirmPay = async () => {
+    if (!selectedFeeId) return;
+    payMonthlyFee({
+      feeId: selectedFeeId,
+      studentId: student.id,
+    })
+      .unwrap()
+      .then(() => {
+        toast.success("Pago realizado correctamente");
+      })
+      .catch(() => {
+        toast.error("Ocurrió un error al procesar el pago");
+      })
+      .finally(() => {
+        setModalType(null);
+        setSelectedFeeId(null);
+      });
+  };
+
+  const handleCancelPay = () => {
+    setModalType(null);
+  };
+  if (isLoading) return <p>Cargando cuotas...</p>;
+  if (isError) return <p>Error al cargar cuotas.</p>;
+
+  const columns: Column<StudentMonthlyFeeResponse>[] = [
+    {
+      header: <SortableButton text="N° de cuota" />,
+      accessor: "number",
+    },
+
+    {
+      header: <SortableButton text="Mes" />,
+      accessor: "month",
+      Cell: ({ original }) => <span>{translateMonth(original.month)} </span>,
+    },
+
+    {
+      header: <SortableButton text={"Fecha de\nvencimiento"} allowWrap />,
+      accessor: "expirationDate",
+      Cell: ({ original }) => <span>{original.expirationDate}</span>,
+    },
+    {
+      header: <SortableButton text="Monto" />,
+      accessor: "amount",
+      Cell: ({ original }) => <span>{formatCurrency(original.amount)}</span>,
+    },
+    {
+      header: "Estado",
+      accessor: "status",
+      Cell: ({ original }) => (
+        <FeeStatusContainer>
+          <FeeStatus $status={original.status}>{original.status}</FeeStatus>
+        </FeeStatusContainer>
+      ),
+    },
+    {
+      header: "Pago",
+      Cell: ({ original }) => {
+        const canPay = ["Pendiente", "Vencida"].includes(original.status);
```

Comment:
Ahora, este status me parece que es "Vencido" no "Vencida"


```suggestion
        const canPay = ["Pendiente", "Vencido"].includes(original.status);
```

Reply:
Ah bien, igual no debería ser vencida? es una cuota vencida, no vencido.

Reply:
En el diseño está como vencida: 
<img width="1150" height="57" alt="image" src="https://github.com/user-attachments/assets/2d1465f2-297e-4b23-97c0-9d0a792be68d" />

Reply:
bueno ahi vi en una pr del back que es vencido: 
<img width="345" height="115" alt="image" src="https://github.com/user-attachments/assets/94852421-7928-4b66-8f67-ddf2e94d076e" />

Reply:
Sip, en el back está como "Vencido", por eso lo decía. Pero coincido que quedaría mejor "vencida", podemos hacer el cambio luego